airflow/settings.py

```diff
@@ -578,6 +578,10 @@ def initialize():
     executor_constants.CELERY_KUBERNETES_EXECUTOR,
     executor_constants.LOCAL_KUBERNETES_EXECUTOR,
 }
+
+# Executors can set this to true to configure logging correctly for
+# containerized executors.
+IS_EXECUTOR_CONTAINER = bool(os.environ.get("AIRFLOW_IS_EXECUTOR_CONTAINER", ""))
```

Comment:
same question as on the `airflow/utils/log/logging_mixin.py`

Reply:
Changes have been extracted into a core change PR and has been merged. Release in 2.8.0